docs/05-plugin-development.md

```diff
@@ -187,6 +187,23 @@ Kind: `async, parallel`
 
 Called initially each time `bundle.generate()` or `bundle.write()` is called. To get notified when generation has completed, use the `generateBundle` and `renderError` hooks.
 
+#### `augmentChunkHash`
+Type: `(preRenderedChunk: PreRenderedChunk) => any`<br>
```

Comment:
Is `any` really the correct type? Looking at how you actually use the return value, the logic assumes these are strings or things that can be safely converted to string, so definitely no functions or objects. I would actually use `string` here to avoid confusion.